edu.cuny.hunter.log.core/src/edu/cuny/hunter/log/core/analysis/LogAnalyzer.java

```diff
@@ -369,13 +384,27 @@ private Boolean checkIfBlock(ASTNode node) {
 						|| condition.contains("DEBUG") || condition.contains("TRACE")) {
```

Comment:
Shouldn't you ignore case here?

Reply:
I ignored. When it returns true, the transformation action will be NONE and the tool will count them.